content/deploy/README.md

```diff
@@ -35,9 +42,127 @@ Events:
+Ideally, using the [GitOps](https://www.weave.works/technologies/gitops/)
+methodology, you only need to register _one_ repository with deployment access
+for your team. This repository contains all of your team's infrastructure as
+code, including your application deployments. That said, it is also possible to
+add each of your application repositories.
+
+Visit the [registration portal](https://deployment.prod-sbs.nais.io/auth/login)
+and follow the instructions to enable deployments from your repository.
+In order to do this, you need to have _maintainer_ access rights to your Github
+team, and _admin_ access to the repository.
+
+### Authenticating to the Github API
+There are two ways to authenticate API requests: using a Github App, or with a personal access token.
+
+The first option is unfortunately currently only available to Github organization admins. You can still self-service by
+[creating a personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)
+that your CI pipeline can use to trigger the deployment. The token needs only the scope `repo_deployment`.
+
+### Performing the deployment
+In your pipeline, use our internal tool [deployment-cli](https://github.com/navikt/deployment-cli)
+to make deployment requests. Variables you need include _environment_, _repository_, _team_, _application version_, _Kubernetes resources_, and your Github credentials.
+
+Example syntax:
+
+```
+deployment-cli create \
+  --environment=dev-fss \
+  --repository=navikt/deployment \
+  --team=<TEAM> \
+  --version=<VERSION> \
+  --appid=1234 \
+  --key=/path/to/private-key.pem \
+  --resources=nais.yaml \
+  --config=placeholders.json
+```
+
+If for any reason you are unable to use _deployment-cli_, please read the section on [NAIS deploy with cURL](#nais-deploy-with-curl).
+
+### Deployment request spec
+
+| Key | Description | Version added |
+|-----|-------------|---------------|
+| environment | Which environment to deploy to. | N/A |
+| payload.version | This is the *payload API version*, as described below. Array of three digits, denoting major, minor, and patch level version. | 1.0.0 |
+| payload.team | Github team name, used as credentials for deploying into the Kubernetes cluster. | 1.0.0 |
+| payload.kubernetes.resources | List of Kubernetes resources that should be applied into the cluster. Your `nais.yaml` file goes here, in JSON format instead of YAML. | 1.0.0 |
+
+### Environment
+Please use one of the following environments. The usage of `preprod-***` is *not* supported.
+  * `dev-fss`
+  * `dev-sbs`
+  * `prod-fss`
+  * `prod-sbs`
+
+### NAIS deploy with cURL
+A deployment into the Kubernetes clusters starts with a POST request to the [GitHub Deployment API](https://developer.github.com/v3/repos/deployments/#create-a-deployment).
+The request contains information about which environment to deploy to, which team to deploy as, and what resources should be applied.
```

Comment:
```suggestion
The request contains information about which cluster to deploy to, which team to deploy as, and what resources should be applied.
```

Reply:
Can we remove the heading manual deploy. I dont want that to be an option

Reply:
I'm not sure I agree it should be completely removed, as this might be relevant in early stages of development. It's nice to provide a basic understanding of what actually happens with the CRD and that it's actually a resource being applied to the cluster. Maybe move it down a bit instead?